### DISCUSSION TITLE
Does ReactiveUI.Events not support .NET 6?

### DISCUSSION THREAD

**Discussion by guilhermeolisi** [reactiveui/ReactiveUI#3133] [General]
Hi,

I'm trying to use ReactiveUI.Events in my project but I get an error installing the nuget package: "The ReactiveUI.Events 15.1.1 is not compatible with .net6.0". Are There plans to make the package compatible with .NET 6?

**Comment by glennawatson** [comment]
We stopped making rxui.events. you can use https://github.com/reactivemarbles/ObservableEvents instead

**Reply by guilhermeolisi** [reply]
Thanks for your replay, i will see this package
